src/write.c

```diff
@@ -536,6 +537,13 @@ static avifBool avifEncoderDetectChanges(const avifEncoder * encoder, avifEncode
     return AVIF_TRUE;
 }
 
+// Subset of avifEncoderWriteColorProperties() for the properties clli, pasp, clap, irot, imir.
+static void avifEncoderWriteExtendedColorProperties(avifRWStream * dedupStream,
```

Comment:
Should the local variable `avifRWStream * s` in `avifEncoderWriteColorProperties()` also be renamed `dedupStream`?

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1503 but not so sure it is better.

Reply:
This is for consistency. I was wondering why that `avifRWStream` variable is named `s` in one function but named `dedupStream` in the other.

I forgot how the code in these two functions works, so I am not sure if `dedupStream` is a good name. It may be better to add comments to describe `dedupStream` vs. `outputStream`.